CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CanHazFunny
+{
+    public class Jester
+    {
+        private IJokeOutput jokeOutput;
+        private IJokeService jokeService;

```

Comment:
Fields should be named with an underscore prefix and PascalCase
```suggestion
        private IJokeOutput _JokeOutput;
        private IJokeService _JokeService;

```